includes/class-wc-tracker.php

```diff
@@ -493,7 +493,7 @@ private static function get_admin_user_agents() {
 	 *
 	 * @return array
 	 */
-	private static function get_order_totals() {
```

Comment:
Is WC_Tracker needed once tracks are implemented?

Reply:
Yes, should keep both.